### DISCUSSION TITLE
Editing Wide Tables is Broken

### DISCUSSION THREAD

**Discussion by firedaemon-ceo** [GitbookIO/community#154] [0. General bugs and questions]
### What's your feature/idea?

If you have a lot of columns in a table it is difficult to edit the table as it is not possible to grab the horizontal scroll bar at the bottom of the table to scroll left and right. You literally have to tab between cells. Then finally it's not possible to resize the last column in a wide table as you can't grab the final column's right resize anchor.

<img width="1272" alt="sshot-1" src="https://user-images.githubusercontent.com/45594173/227154817-eedad8e2-44fd-437f-8d96-de853d828477.png">

Can replicate on the latest Chrome, FF, and Edge on Windows 11.

### What's your desired outcome?

Wide table editing to work properly.

### What's the impact of this for you?

- [X] Critical
- [ ] High
- [ ] Medium
- [ ] Low

**Comment by taszap** [comment]
Hi @firedaemon-ceo thank you for your post. 
We are still quite a small team so we are not able to offer speedy technical support via the community. If you encounter any bugs or issues, we strongly encourage you to contact [support@gitbook.com](mailto:support@gitbook.com) so we can investigate this for you. 

Would you be able to please email us and we will look into it?
